content/attribute_precedence.md

```diff
@@ -41,7 +41,14 @@ precedence:
 
 Attribute precedence, when viewed as a table:
 
-![image](/images/overview_chef_attributes_table.png)
+|                | Attribute Files | Node/Recipe | Environment| Role |
+|----------------|-----------------|-------------|------------|------|
+| default        | 1               | 2           | 3          | 4    |
+| force_default  | 5               | 6           |            |      |
+| normal         | 7               | 8           |            |      |
+| override       | 9               | 10          | 12         | 11   |
+| force_override | 13              | 14          |            |      |
+| automatic      |                 |             | 15         |      |
```

Comment:
Unfortunately it's gotta be HTML to get the colspan in the last row.

```suggestion
<table>
<thead>
<tr>
<th>Attribute Type</th>
<th>Attribute Files</th>
<th>Node/Recipe</th>
<th>Environment</th>
<th>Role</th>
</tr>
</thead>
<tbody>
<tr>
<td>default</td>
<td>1</td>
<td>2</td>
<td>3</td>
<td>4</td>
</tr>
<tr>
<td>force_default</td>
<td>5</td>
<td>6</td>
<td></td>
<td></td>
</tr>
<tr>
<td>normal</td>
<td>7</td>
<td>8</td>
<td></td>
<td></td>
</tr>
<tr>
<td>override</td>
<td>9</td>
<td>10</td>
<td>12</td>
<td>11</td>
</tr>
<tr>
<td>force_override</td>
<td>13</td>
<td>14</td>
<td></td>
<td></td>
</tr>
<tr>
<td>automatic</td>
<td style="text-align:center" colspan="4">15</td>
</tr>
</tbody>
</table>
```

Reply:
Im sorry, Im not sure I follow. What is the colspan needed for here?